app/utils/add_sheets_items.py

```diff
@@ -11,314 +10,132 @@
 )
 
 # data sheets objects. bot_sheet -> already existing bot data andela_sheet -> Andela equip data sheet
-andela_sheet = gc.open("Asset Tracker For bot").worksheet("Master  Inventory List")
-
+andela_sheet = gc.open('Benjamin Copy of Asset Tracker For bot').worksheet("Master  Inventory List")
```

Comment:
Given that you're doing this same thing in `get_all_items` I don't think we need this line